### ISSUE TITLE
Accessibility - tabindex values of -1 used for key navigation links

### ISSUE THREAD

**Issue by gregdorward** [redbadger/website-honestly#368]
**Description of the issue:**
We currently have a tabindex value of -1 set for the following links on the homepage:
- Home
- About us
- What we do
- Blog
- Events
- Jobs
- Contact us

**Browser used:**
Chrome

**OS used:**
OSX El Capitan

**Device used:**
Macbook Pro

**User Impact**
On some browsers, a user that has to use a keyboard only will not be able to tab to these elements

**Steps to reproduce:**

1. View the page source for the Red Badger Homepage
2. Search for "tabindex"
3. View the results

**Actual result:**
The elements listed above all have a tabindex value of -1 

**Expected result:**
These elements should not have this value, for the reasons outlined by WAVE, the web accessibility evaluation tool:

> **Why It Matters**
> 
> Tabindex can facilitate keyboard navigation for interactive elements. A tabindex attribute value of 0 places an item into the keyboard navigation order (i.e., you can navigate to it using the Tab key). A value of less than 0 (typically -1) removes an element from the keyboard flow (you cannot Tab to it), but allows it to receive programmatic focus (e.g., via scripting).
> 
> **How to Fix It**
> 
> Ensure that tabindex is being used correctly by navigating and interacting with the elements using only the keyboard. Positive tabindex values specify a distinct tab order and should typically be avoided

**Further reading**

I'm not sure how best to resolve this one, as I'm guessing these values have been assigned for a reason, but there is some guidelines available here - https://adamsoucie.com/how-to-use-tab-index-accessibly/

**Comment by asavin** [comment]
@gregdorward those are most likely the ones on the mobile nav - they are disabled on purpose so that they are not displayed in the list of links on screen reader. Otherwise you will get duplicated of the same links, as we are also having desktop navigation.

**Comment by gregdorward** [comment]
Ah yes, I think you're right, good spot. I'll close this as it's not a genuine issue then
